plane/source/docs/thermal-mode.rst

```diff
@@ -0,0 +1,16 @@
+.. _thermal-mode:
+
+============
+THERMAL Mode
+============
+
+If SOARING is enabled (:ref:`SOAR_ENABLE<SOAR_ENABLE>` = 1 and it has not been disabled by an RC switch (``RCx_OPTION =88``), this mode will idle the throttle, and begin circling searching for vertical lift.
+Navigation is controlled in the same manner as the previous mode, ie if a pilot controlled mode, it remains so. If an auto-navigating mode, it continues the course demanded by that mode. 
+
+This is the mode that is used for autonomous soaring (See :ref:`soaring-4_1`) and is automatically entered if a thermal is detected during the gliding portion of SOARING flight, and will begin circling searching for the lift center.
+
+Like the THERMAL mode phase of autonomous SOARING it will exit under the same conditions and return to the previous flight mode.
```

Comment:
This makes it sound like there are two THERMAL modes. Suggest "As described under autonomous soaring, there are conditions under which the previous flight mode may be restored"

Reply:
done